src/test/java/com/hazelcast/aws/DescribeInstancesTest.java

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
i don't know why you recreated this file in another package. That makes it a bit hard to review the changes..